clients/web/src/lib/components/atoms/Modal.svelte

```diff
@@ -7,36 +7,36 @@
 
     export let title: string;
     export let visible: boolean = false;
-    export let canClickOutside: boolean = false;
```

Comment:
Why remove this prop?

Reply:
It wasn't in use. So I thought it was better coding to clear it out. It was showing grey in the IDE. I was thinking like an unused import. 

The only time handle clickoutside can be called is if you are clicking outside the modal. Then you don't need to track they in that file. You just set visible to false and it seemed to run the remove part of the click outside. 

It was how the svelte tutorial did it but I am probably missing something 😅

Reply:
This prop was in place such that a modal might not let you click outside to close (i.e. it may have some required stuff a user must do)

Reply:
We should restore it